assets/src/design-system/components/search/test/input.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
+import { fireEvent } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { renderWithProviders } from '../../../testUtils/renderWithProviders';
+import SearchInput from '../input';
+
+describe('Search <Input />', () => {
+  const onClickMock = jest.fn();
+  const handleClearInputMock = jest.fn();
+
+  beforeEach(jest.clearAllMocks);
+
+  it('should render a <SearchInput /> combobox by default', () => {
+    const { getByRole } = renderWithProviders(
+      <SearchInput
+        ariaInputLabel={'search label'}
+        ariaClearLabel={'aria label for clearing value'}
+        onClick={onClickMock}
+        handleClearInputValue={handleClearInputMock}
+        inputValue={''}
```

Comment:
💭 We don't need string interpolation here! I know our storybook linting likes it so those files confuse me 🤔 
```suggestion
        ariaInputLabel="search label"
        ariaClearLabel="aria label for clearing value"
        onClick={onClickMock}
        handleClearInputValue={handleClearInputMock}
        inputValue=""
```

Reply:
thanks! i _think_ i got them all.